api/src/main/java/io/cloudevents/types/Time.java

```diff
@@ -38,16 +38,16 @@ private Time() {
         .toFormatter();
 
     /**
-     * Parse a {@link String} RFC3339 compliant as {@link ZonedDateTime}
+     * Parse a {@link String} RFC3339 compliant as {@link OffsetDateTime}
      */
-    public static ZonedDateTime parseTime(String time) throws DateTimeParseException {
-        return ZonedDateTime.parse(time, RFC3339_DATE_FORMAT);
+    public static OffsetDateTime parseTime(String time) throws DateTimeParseException {
+        return OffsetDateTime.parse(time, RFC3339_DATE_FORMAT);
```

Comment:
The default formatter used by `OffsetDateTime` is `DateTimeFormatter.ISO_OFFSET_DATE_TIME`, which conforms exactly to the RFC3339 "date-time" specification.  It is therefore unnecessary to use a custom formatter;
```
    OffsetDateTime.parse(time);
```
will achieve the same result more simply and clearly.

Similarly,
```
    offsetDateTimeValue.toString();
```
will output `OffsetDateTime` values in RFC3339 form without the need to specify a formatter.  The Jackson Java 8 Date/time module also uses this format by default for `OffsetDateTime`.  (This is one of the reasons for recommending the change from `ZonedDateTime` to `OffsetDateTime`.)

Reply:
lemme try :smile: